transformer/node_dispatcher.ts

```diff
@@ -41,55 +32,53 @@ const isMethodCall = (node: ts.Node, method: string): boolean => {
     }
 };
 
-// Determine if given node is RxJS Pipe Statement.
-const isPipeStatement = (node: ts.Node): boolean => isMethodCall(node, 'pipe');
+// Check if node is pipe property access expression.
+const isPipePropertyAccessExpr = (node: ts.Node): boolean => {
+    if (ts.isCallExpression(node) && ts.isPropertyAccessExpression(node.expression)) {
+        return node.expression.name.getText() === 'pipe' ? true : false;
```

Comment:
=== operator evaluates to a boolean
`return node.expression.name.getText() === 'pipe'` is the same